### ISSUE TITLE
the environment variables cannot contain "-" error in azure container

### ISSUE THREAD

**Issue by pepepaco** [japtain-cack/bedrock-server#2]
I just cant figure out how to set the level-seed while using azure containers, they restart and overwrite the server.properties after I try to change anything,
I know it wasn't intended to work in azure containers so for now I can only use default seed.
other than that it works great. thank you.

**Comment by japtain-cack** [comment]
This is a typo, should be an `_`. I will fix.

**Comment by japtain-cack** [comment]
You are correct that some platforms don't allow `-` in environment variables. This works on many systems but is a bad practice. An oversight on my part.

**Comment by japtain-cack** [comment]
https://github.com/japtain-cack/bedrock-server/pull/3

**Comment by japtain-cack** [comment]
Please let me know if that resolves your issue.
